Syllabus.md

```diff
@@ -0,0 +1,116 @@
+reproducible research. This course emphasizes direct, hands-on experience with
+applications to current biological research problems. The course is **NOT
+INTENDED** for computer science students who want to learn about biologically
+motivated algorithmic problems; GCB/CIS/BIOL536 and GCB537 is more appropriate.
+
+Areas include DNA sequence alignment, genetic variation and analysis, motif
+discovery, study design for high-throughput sequencing, RNA and gene expression,
+single gene and whole-genome analysis, machine learning, and topics in systems
+biology. The relevant principles underlying methods used for analysis in these
+areas will be introduced and discussed at a level appropriate for biologists
+without a background in computer science.
+
+The course will assume a solid knowledge of modern biology. An advanced
+undergraduate course such as BIOL421 or a graduate course in biology such as
+BIOL526 (Experimental Principles in Cell and Molecular Biology), BIOL527
+(Advanced Molecular Biology and Genetics), BIOL528 (Advanced Molecular
+Genetics), BIOL540 (Genetic Systems), or equivalent, is a prerequisite.
+
+## Equipment prerequisite
+
+All students are required to bring a laptop to class for in class activities.
+TAs will provide help with the material, but students should be computer-capable
+with their own laptop, and should be willing/capable to download and install
+free software from the Internet.
+
+## Grading
+
+### Composition of grades
+
+Grades are composed of:
+
+* the course project (50% total, 25% written proposal,
+25% oral presentation)
+* homework assignments (30%)
+* in-class labs assignments (10%)
+* class participation (10%).
+
+For the homework grade, we will drop scores from the 3 lowest scoring
+assignments when determining the final grade.
+
+### Late grading policy:
+
+For in-class lab assignments, we will not accept late turn-ins. For homework, we
+will grade assignments turned in after the due date up to one week past the
+deadline, but grades will be penalized (minus 30% off). Our late policy is
+driven by fairness, because answer keys will be handed out a week after problems
+are due.
+
+## Plagiarism policy
+
+Consistent with the University of Pennsylvania's honor code and policies on
+academic integrity, we maintain a zero-tolerance policy on plagiarism. For
+assignments containing text that the instructors determine is plagiarized,
+students will receive a grade of zero for the assignment and the case will be
+referred to the Office of Student Conduct for disciplinary measures. Students
+may not always be aware of what constitutes plagiarism in their work. If you are
+unsure, please contact one of the course directors. Please see links below
+regarding the office of academic integrity for Penn’s policy on plagiarism and
+our discretion on grading.
+
+https://provost.upenn.edu/policies/faculty-handbook/students/iv-d
+http://www.upenn.edu/academicintegrity/ai_violations.html
+
+## Reference Texts
+
```

Comment:
While, as a contributor to one of the PLOS CB collections, I view it as a critically important resource, I do wonder if this is a helpful set of resources. We do have connections to other resources within the assignments. I wonder if that obviates the need for this section of the syllabus.